### DISCUSSION TITLE
hi. how to backup tags?

### DISCUSSION THREAD

**Discussion by leonalux** [stashapp/stash#3612] [Q&A] [answered]
I used a lot of tags, how do I save them? 
I found many functions in settings-system, but I don't know which one is tag save function. 
I will be sad if tags are lost.

and many thanks for made this awesome app.

**Comment by ALonelyJuicebox** [comment, marked as answer]
The Tags themselves are saved in your Stash database once created. You can make a backup of your Stash database from the Settings area (click on the gear icon on the upper right) and under the Tasks area, click on the "Download Backup" button.

**Reply by leonalux** [reply]
many thx!
